packages/workspace/src/browser/i18n.spec.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import fs = require('fs');
+import "mocha";
+import * as chai from "chai";
+const Globalize = require("globalize");
+
+const expect = chai.expect;
+
+beforeEach(() => {
```

Comment:
I don't mind comments and such much in test files, but could that beforeEach hook be removed if it's not doing anything?